objects/authentication_factor.json

```diff
@@ -0,0 +1,85 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
+      "enum": {
+        "sms": {
```

Comment:
By convention, enums are integers I think -- see some of the examples of enums in existing objects.